boba-bindings/gen_bindings.sh

```diff
@@ -0,0 +1,59 @@
+#/bin/bash
```

Comment:
Same here, this bash has disappeared from upstream, so maybe good to move to the newer variant.